AlphaWallet/Common/Services/FirebaseReportService.swift

```diff
@@ -15,14 +15,16 @@ extension AlphaWallet {
         //NOTE: to avoid warning `The default Firebase app has not yet been configured. FirebaseApp.configure()`, moving code to init method have no affect
         static var instance: FirebaseCrashlyticsReporter = {
             if let options = R.file.googleServiceInfoPlist().flatMap({ FirebaseOptions(contentsOfFile: $0.path) }) {
-                if !isRunningTests() && isAlphaWallet() {
+                if isAlphaWallet() {
                     FirebaseApp.configure(options: options)
                 }
             }
             return FirebaseCrashlyticsReporter()
         }()
 
-        private init() { }
+        private init() {
+            crashlytics.setCrashlyticsCollectionEnabled(!isRunningTests())
```

Comment:
Not sure it works correctly because the function's comment is:

```
 * The value does not apply until the next run of the app. If you want to disable data
 * collection without rebooting, add the FirebaseCrashlyticsCollectionEnabled flag to your app's
 * Info.plist.
```

If it's too much of a hassle, let's just leave it. It's only printing the warning when running tests right?

Reply:
Updated pr with plist for tests, and disabled collection in plist